me.brokenby.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.brokenby.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - '/app/bin/pmedit-cli'
```

Comment:
The command doesn't match the desktop file Exec?

Reply:
command is of type "string" not "array". Flatpak-builder will, at best, complain if not simply bail out. Did you build this locally?